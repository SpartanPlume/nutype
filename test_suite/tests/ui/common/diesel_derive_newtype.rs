use nutype::nutype;

#[nutype(derive(Debug, DieselNewType))]
pub struct DieselNewTypeString(String);

fn main() {}
