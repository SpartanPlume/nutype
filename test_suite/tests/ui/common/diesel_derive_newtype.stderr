error: To derive DieselNewType, the feature `diesel-derive-newtype` of the crate `nutype` needs to be enabled.
 --> tests/ui/common/diesel_derive_newtype.rs:3:24
  |
3 | #[nutype(derive(Debug, DieselNewType))]
  |                        ^^^^^^^^^^^^^
